Structure real_building
Structure grid_building
